src/Tools/dotnet-monitor/CollectionRules/Options/Actions/LoadProfilerOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Diagnostics.Monitoring.WebApi.Models;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Actions
+{
+    /// <summary>
+    /// Options for the <see cref="CollectionRules.Actions.LoadProfilerAction"/> action.
+    /// </summary>
+    [DebuggerDisplay("LoadProfiler")]
+    internal sealed partial class LoadProfilerOptions
+    {
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_LoadProfilerOptions_Path))]
+        [Required]
+        public object Path { get; set; } = new Dictionary<string, string>();
```

Comment:
Should we be defining this as an object? My understanding is we're doing it for flexibility to be able to assign it to either a string or a dictionary, and then `PathMap` takes the `Path` (if it's a string) and pushes it into that dictionary format, but couldn't we just make `Path` a dictionary and force all assignments to be in the form of a dictionary (even if there's only a single entry)? I could be overlooking a good reason for taking this approach though.